ui/src/utils/env.ts

```diff
@@ -0,0 +1,47 @@
+export {}
```

Comment:
what's this export doing?

Reply:
NO IDEA.. but it was in the other webpack configs. @121watts have any insight here?